paasta_tools/kubernetes_tools.py

```diff
@@ -1460,6 +1460,8 @@ def get_kubernetes_containers(
         secret_volumes: Sequence[SecretVolume],
         service_namespace_config: ServiceNamespaceConfig,
         include_sidecars: bool = True,
+        include_liveness_probe: bool = True,
+        include_readiness_probe: bool = True,
```

Comment:
i'd consider making this a single `include_probes` since it's very likely that we only ever want none or all of 'em - but this is a pretty easy refactor down the line so it's fine :)